packages/core/src/extension/resizable-iframe-creator.ts

```diff
@@ -58,16 +68,27 @@ export default class ResizableIframeCreator {
 
                     const iframes = iframeResizer(
                         {
+                            autoResize: false,
                             scrolling: false,
                             sizeWidth: false,
-                            heightCalculationMethod: 'bodyScroll',
+                            heightCalculationMethod: 'bodyOffset',
                         },
                         iframe,
                     );
 
-                    teardown();
                     resolve(iframes[iframes.length - 1]);
                 }
+
+                if (
+                    typeof event.data === 'string' &&
+                    /\[iFrameSizer]iFrameResizer.*init/.test(event.data)
```

Comment:
Do you know all the possible values of `event.data`? It's better to list them out explicitly rather than using regexp.

Reply:
The `event.data` format is a string, `\[iFrameSizer\]iFrameResizer[seq of the iframe]:[height]:[width]:init`.
`init` can be other values, such as `size`, `reset`. 
To confirm the successful initialization of `iframeResizer`, we just need to look for `init`.

Reply:
Oh okay interesting. I didn't know about that. Does `iframe-resizer` provide a public API to listen for the `init` event? For example, it seems there's `onInit` callback that we can use? (https://github.com/davidjbradshaw/iframe-resizer/blob/master/docs/parent_page/events.md#oninit)

Reply:
`initCallback` can be used as a `iframeResizer`'s initialization option.
As a result, we now rely on `iframeResizer` to notify an extension when it is ready.